packages/bsky/src/db/tables/post.ts

```diff
@@ -12,6 +12,8 @@ export interface Post {
   replyParent: string | null
   replyParentCid: string | null
   langs: string[] | null
+  isInvalidReply: Generated<boolean>
+  isInvalidInteraction: Generated<boolean>
```

Comment:
This "interaction" terminology is a bit leftover from the last pass on this, I wonder if we should change it.  The `isInvalidReply` means that the contents of `post.reply` just doesn't look accurate, e.g. the reply parent and root aren't compatible.  The `isInvalidInteraction` incorporates the idea of violating a thread's reply/interaction gate. 🤔

Reply:
i agree - it's not immediately clear from reading...

maybe just making it super clear

- `invalidReplyRoot`
- `violatesThreadGate`